Kickstarter-iOS/Features/PledgeOverTime/Views/Cells/PledgePaymentPlanCell.swift

```diff
@@ -0,0 +1,171 @@
+import Library
+import UIKit
+
+final class PledgePaymentPlanCell: UITableViewCell, ValueCell {
+  // MARK: properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+
+  private lazy var leftColumnStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var rigthColumnStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var titleLabel = { UILabel(frame: .zero) }()
+  private lazy var subtitleLabel = { UILabel(frame: .zero) }()
+  private lazy var checkmarkImageView: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var selectionView: UIView = { UIView(frame: .zero) }()
+
+  private let viewModel: PledgePaymentPlansCellViewModelType = PledgePaymentPlansCellViewModel()
+
+  // MARK: Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.configureSubviews()
+    self.setupConstraints()
+    self.bindViewModel()
+  }
+
+  @available(*, unavailable)
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Configuration
+
+  private func configureSubviews() {
+    self.contentView.addSubview(self.rootStackView)
+
+    addArrangedSubviews([self.checkmarkImageView, UIView()], to: self.leftColumnStackView)
+
+    addArrangedSubviews([self.titleLabel, UIView()], to: self.rigthColumnStackView)
```

Comment:
Great question! The `UIView()` here acts as a flexible spacer to prevent the `UIImageView` from stretching when the `UIStackView` distributes its subviews. Without it, the `UIImageView` might resize in ways that don’t align with the intended design. Let me know if you’d like me to elaborate further or if you think there’s a better way to handle this!

Reply:
That's creative! I would personally not use stack views for the cell itself if it requires hacks like this (just use constraints, especially for the image!). But I'll leave the final decision up to you here